src/modules/menu/MenuModule.vue

```diff
@@ -40,7 +40,7 @@
                     class="menu-tray"
                     :class="{
                         'desktop-mode': isDesktopMode,
-                        'desktop-menu-closed': isDesktopMode && !showMenu,
+                        'desktop-menu-closed': isDesktopMode && !isMenuShown,
```

Comment:
Don't know if `isMenuShown` will ever be false on desktop (yet, as no fullscreen support on desktop implemented for now)